app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class= "welcome-wrapper">
+  <h3>
+    <span id="spotlight">Media Spotlight:</span>
+    <%= link_to @spotlight.title, work_path(@spotlight)%>
+    by <%= @spotlight.creator %>
+  </h3>
+  <span id="spotlight-votes"><%= pluralize(@spotlight.votes.count, "vote") %></span>
+  <%= @spotlight.description %>
+</p>
+
+<hr />
+
+<div class="grid-x">
+
+<section class="top-ten small-4 columns">
+  <h3>Top Albums</h3>
+  <ul>
+    <% @albums.each do |work| %>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?